specs/006-fix-dashboard-data/tasks.md

```
@@ -0,0 +1,329 @@
+- **Foundational (Phase 2)**: Depends on Setup (T001) — BLOCKS all user stories
+- **US1 Aggregates (Phase 3)**: Depends on Foundational completion (T004-T022)
+- **US2 Gaps & Threads (Phase 4)**: Depends on Foundational completion — can run in PARALLEL with US1
+- **US3 Exceptions (Phase 5)**: Depends on Foundational completion — can run in PARALLEL with US1, US2
+- **US4 Filters (Phase 6)**: Depends on Foundational completion — can run in PARALLEL with US1-US3
+- **US5 Time Series (Phase 7)**: Depends on Foundational completion — can run in PARALLEL with US1-US4
+- **US6 Distribution (Phase 8)**: Depends on US1 completion (needs aggregate data for distribution generation)
+- **Polish (Phase 9)**: Depends on all user stories being complete
+
+### User Story Dependencies
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)